packages/@aws-cdk/aws-ecs-patterns/lib/base/network-load-balanced-service-base.ts

```diff
@@ -86,6 +86,15 @@ export interface NetworkLoadBalancedServiceBaseProps {
    */
   readonly loadBalancer?: NetworkLoadBalancer;
 
+  /**
+   * Listener port of the network load balancer that will serve traffic to the service.
+   *
+   * [disable-awslint:ref-via-interface]
+   *
+   * @default - 80
+   */
+  readonly listenerPort?: number;
```

Comment:
Is there any reason why this wouldn't apply to the ApplicationLoadBalancedServiceBase as well?

Reply:
support ApplicationLoadBalancedServiceBase now